roles/start-services/tasks/main.yaml

```diff
@@ -5,3 +5,10 @@
     chdir: "{{ e2dc_install_path }}"
   become: yes
   become_user: "{{ stereum_user }}"
+  when: setup != "multiclient"
+
+- name: Start services multiclient
```

Comment:
this is doing the same as the task above

Reply:
got it